ObolNetwork/obol-site
# Test uploading terms as a static file rather than dropbox link

### Summary

Swaps in new t's&c's

### Details

### How to test it
Check how much bigger the bundle size is and see if this is an issue to us (presume not on gh pages because we don't pay for it directly)

- [ ] Successful deployment to gh-pages

### Screenshots

TBC

ticket: #000
